src/app/work-item/work-item-list/work-item-list.component.ts

```diff
@@ -1,19 +1,19 @@
-import { 
+import {
```

Comment:
Actually I made the changes in this file but remove it from this PR. 
In my code editor once I save any file then it will remove all extra white spaces. That's why only white space removal changes is here.